src/main/java/de/retest/recheck/review/ignore/io/Loader.java

```diff
@@ -1,11 +1,21 @@
 package de.retest.recheck.review.ignore.io;
 
+import java.util.Optional;
+import java.util.stream.Stream;
+
 public interface Loader<T> {
 
-	boolean canLoad( final String line );
```

Comment:
I should have removed all `canLoad` methods except from `FilterPreserveLineLoader` since this has a more complex `canLoad` method. However, I forgot to make it private, which I have now done.